core/mock_test.go

```diff
@@ -28,9 +28,11 @@ func CheckErrors(t *testing.T, err error, expected error) {
 type mockEC2 struct {
 	ec2iface.EC2API
 
-	// Describe Spot Price History
-	dspho   *ec2.DescribeSpotPriceHistoryOutput
-	dspherr error
+	// DescribeSpotPriceHistoryPages output
+	dsphpo []*ec2.DescribeSpotPriceHistoryOutput
```

Comment:
Rename `dspho` to `dsphpo` and make it an array of outputs. Each element in the array corresponds to a page.